<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UE4 Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="welcomeLabel">
      <property name="text">
       <string>Welcome to the unofficial Unreal Engine 4 Manager for GNU/Linux!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="installButton">
      <property name="text">
       <string>Install Unreal Engine 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updateButton">
      <property name="text">
       <string>Update Unreal Engine 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openMarketplaceButton">
      <property name="text">
       <string>Open the Marketplace</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="marketplaceDownloaderButton">
      <property name="text">
       <string>Download content from the Marketplace</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="launchButton">
      <property name="text">
       <string>Launch Unreal Engine 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
